ruff.toml

```diff
@@ -30,21 +30,17 @@ extend-select = [
 
 	# local
 	"ANN2", # missing-return-type-*
-	"F", # Pyflakes
```

Comment:
The `"F"` rule is not present in skeleton.
Should I assume that it is automatically enabled by `ruff`?

Reply:
No need to assume, here's the reference! 
- https://docs.astral.sh/ruff/linter/#rule-selection
- https://docs.astral.sh/ruff/settings/#lint_extend-select